Managers/LuaMan.cpp

```diff
@@ -871,6 +872,7 @@ int LuaMan::Initialize() {
 			.def_readonly("Attachables", &MOSRotating::m_Attachables, return_stl_iterator)
 			.def_readonly("Wounds", &MOSRotating::m_Wounds, return_stl_iterator),
 
+
```

Comment:
Can you undo the whitespace you added throughout the file?

Reply:
Can't tell exactly why this got added, but it seems merging `dev` solved it. They're gone now.